cmd/updater/command/command.go

```diff
@@ -56,6 +59,7 @@ Datadog Updater updates your agents based on requests received from the Datadog
 	agentCmd.PersistentFlags().StringVarP(&globalParams.ConfFilePath, "cfgpath", "c", "", "path to directory containing updater.yaml")
 	agentCmd.PersistentFlags().StringVarP(&globalParams.Package, "package", "p", "", "package to update")
 	agentCmd.PersistentFlags().StringVarP(&globalParams.RepositoriesDir, "repositories", "d", "/opt/datadog-packages", "path to directory containing repositories")
+	agentCmd.PersistentFlags().StringVarP(&globalParams.RunPath, "runpath", "r", "/var/run/datadog-packages", "path to directory containing repositories")
```

Comment:
this should be in the configuration, you can keep it hardcoded for now like it was for the root path (introduced in the next PR) 😅 .

It looks like I forgot to remove the line above in my original PR after it was no longer used